build.sh

```diff
@@ -0,0 +1,121 @@
+
+#!/bin/bash
+
+function usage {
+    echo -e "Usage:"
+    echo -e "$0 CMD"
+    echo -e "Available CMDs:"
+    echo -e "\tvp2420 - build Protectli VP2410 coreboot image"
+    exit 1
+}
+
+if [ $# -le 0 ]; then
+    usage
+fi
+
+function buildImage {
```

Comment:
This function is not even used...

Reply:
Leftover from copying the script. Removed.